Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM java:8-jre
+FROM alpine:3.2
```

Comment:
FYI, I've been playing around with this at home and had issues with ca-certificates when installing openjdk8 and alpine like this. I'm using Maven so it's a different use-case, but something to watch out for. Fixed it by using [oberthur/docker-alpine-java](https://hub.docker.com/r/oberthur/docker-alpine-java/~/dockerfile/) as a base image instead as seen [here](https://github.com/therockstorm/rtn-search/blob/master/Dockerfile).